tests/end-to-end/backstop.js

```diff
@@ -39,59 +39,65 @@ const viewports = [
     hoverSelector: '.navbar-item.has-dropdown.is-hoverable',
     viewports: viewports.filter((viewport) => ['lg', 'xl', 'mx'].includes(viewport.label))
   },
-  { space: 'kyanite-visual-tests', page: 'navbar-megadropdown', selectors: [selectors.body],
+  { space: space, page: 'navbar-megadropdown', selectors: [selectors.body],
     clickSelectors: ['.navbar-burger', '.navbar-item.has-dropdown'],
     viewports: viewports.filter((viewport) => ['md-mini', 'md'].includes(viewport.label)),
     postInteractionWait: 1000,
   },
-  { space: 'kyanite-visual-tests', page: 'navbar-megadropdown-columns', selectors: [selectors.body],
+  { space: space, page: 'navbar-megadropdown-columns', selectors: [selectors.body],
     hoverSelector: '.navbar-item.has-dropdown.is-hoverable',
     viewports: viewports.filter((viewport) => ['lg', 'xl', 'mx'].includes(viewport.label))
   },
-  { space: 'kyanite-visual-tests', page: 'blog-article-header/blog-article-page', selectors: [selectors.container] },
-  { space: 'kyanite-visual-tests', page: 'blog-listing', selectors: [selectors.container] },
-  { space: 'kyanite-visual-tests', page: 'blog-article-author-bio', selectors: [selectors.content] },
-  { space: 'kyanite-visual-tests', page: 'code-snippet', selectors: ['.code-snippet'] },
-  { space: 'kyanite-visual-tests', page: 'hero', selectors: ['.hero'] },
-  { space: 'kyanite-visual-tests', page: 'message', selectors: [selectors.container] },
-  { space: 'kyanite-visual-tests', page: 'table-of-content/table-of-content' },
-  { space: 'kyanite-visual-tests', page: 'card-full-height-variant', selectors: [selectors.body] },
-  { space: 'kyanite-visual-tests', page: 'navbar-external-links', selectors: [selectors.body],
-    label: 'external-link-meganav',
+  { space: space, page: 'blog-article-header/blog-article-page', selectors: [selectors.container] },
+  { space: space, page: 'blog-listing', selectors: [selectors.container] },
+  { space: space, page: 'blog-article-author-bio', selectors: [selectors.content] },
+  { space: space, page: 'code-snippet', selectors: ['.code-snippet'] },
+  { space: space, page: 'hero', selectors: ['.hero'] },
+  { space: space, page: 'message', selectors: [selectors.container] },
+  { space: space, page: 'table-of-content/table-of-content' },
+  { space: space, page: 'card-full-height-variant', selectors: [selectors.body] },
+  { space: space, page: 'navbar-external-links', selectors: [selectors.body],
+    label: `${space}:external-link-meganav`,
     clickSelector: '.navbar-item.has-dropdown.is-hoverable',
     viewports: viewports.filter((viewport) => ['lg', 'xl', 'mx'].includes(viewport.label)),
   },
-  { space: 'kyanite-visual-tests', page: 'navbar-external-links', selectors: [selectors.body],
-    label: 'external-link-dropdown-menu',
+  { space: space, page: 'navbar-external-links', selectors: [selectors.body],
+    label: `${space}_external-link-dropdown-menu`,
```

Comment:
In the case above you use `:` to separate the space from the scenario name.  Here you are using the `_`. Can we have one version?

Reply:
Done